ds/data/processed/hebci/ce_cpd_learner_days.lep.csv

```diff
@@ -0,0 +1,189 @@
+year,lep_id,lep_year_spec,ce_cpd_learner_days
+2014,E37000001,2014,2613000.0
```

Comment:
(5) if these are supposed to be `int`, I'd round them

Reply:
Good catch they are actually floats as you can have half days

Reply:
After checking, this isn't true so have converted to int